packages/asset-service/src/generateAssetData/Utils.test.ts

```diff
@@ -0,0 +1,42 @@
+import { BaseAsset } from '@shapeshiftoss/types'
+
+import { filterBlacklistedAssets } from './generateAssetData'
+
+jest.mock(
+  './blacklist.json',
+  () => [
+    'bip122:000000000019d6689c085ae165831e93/slip44:0',
+    'eip155:1/erc20:0x426ca1ea2406c07d75db9585f22781c096e3d0e0'
+  ],
+  { virtual: true }
+)
+
+const BtcAsset = {
```

Comment:
please use mock asset data that is correctly typed from https://github.com/shapeshift/lib/blob/main/packages/asset-service/src/service/AssetServiceTestData.ts#L140

separate them out into named assets, and recombine them into the same `mockAssets` array.